.github/workflows/build.yml

```diff
@@ -158,7 +158,7 @@ jobs:
       matrix:
         os:
           - ubuntu-24.04
-          - macos-12
+          - macos-14
```

Comment:
Why not 15?

Reply:
It's still in beta https://github.com/actions/runner-images?tab=readme-ov-file#available-images, preferred to be on stable for now.